libkbfs/file_data_test.go

```diff
@@ -24,9 +24,10 @@ func setupFileDataTest(t *testing.T, maxBlockSize int64,
 	maxPtrsPerBlock int) (*fileData, BlockCache, DirtyBlockCache, *dirtyFile) {
 	// Make a fake file.
 	ptr := BlockPointer{
-		ID:      kbfsblock.FakeID(42),
-		KeyGen:  1,
-		DataVer: 1,
+		ID:         kbfsblock.FakeID(42),
+		KeyGen:     1,
+		DataVer:    1,
```

Comment:
Well it depends on the size of the tree I guess.  But this test doesn't actually use this field at all, so I'm just going to delete it in both places so it's not confusing.